app/controllers/concerns/tasklist_ab_testable.rb

```diff
@@ -0,0 +1,72 @@
+module TasklistABTestable
+  TASKLIST_DIMENSION = 66
+
+  TASKLIST_PRIMARY_PAGES = %w(
+    /legal-obligations-drivers-riders
+    /driving-lessons-learning-to-drive/
+    /government/publications/car-show-me-tell-me-vehicle-safety-questions
+    /theory-test/revision-and-practice
+    /theory-test/what-to-take
+    /pass-plus
+  ).freeze
+
+  TASKLIST_SECONDARY_PAGES = %w(
+    /driving-eyesight-rules
```

Comment:
/driving-eyesight-rules is now a primary page as per https://docs.google.com/spreadsheets/d/1A6pWxHloiMvExtiQNmL-8m-ruXQFRluNObK28uhU1jI/edit#gid=1049909139